### ISSUE TITLE
unit tests failing: IndyPayments.ParseGetPaymentSourcesAsync cannot parse response

### ISSUE THREAD

**Issue by jeffallen** [hyperledger/aries-framework-dotnet#36]
I'm currently trying to update from AgentFramework.* namespace to Hyperledger.Aries.* namespaces. I'm filing this issue because while trying to do that, I also have failing unit tests, and I don't know if that's a current known problem or if I've somehow broken something.

On MacOS with tip on 3a915cae803, 11 of the unit tests for Hyperledger.Aries.sln fail.
I am also using libsovtoken @ 1813b3d8b and indy-sdk@  e7bbfdcb2.

An example failure is:

```
Hyperledger.Indy.InvalidStructureException : A value being processed is not valid.
Stack trace:
   at Hyperledger.Aries.Payments.SovrinToken.SovrinPaymentService.RefreshBalanceAsync(IAgentContext agentContext, PaymentAddressRecord paymentAddress) in /Users/jallen/work/sovrin/aries-framework-dotnet/src/Hyperledger.Aries.Payments.SovrinToken/SovrinPaymentService.cs:line 117
   at Hyperledger.Aries.Tests.Payments.TransferTests.MintSovrinTokens() in /Users/jallen/work/sovrin/aries-framework-dotnet/test/Hyperledger.Aries.Tests/Payments/TransferTests.cs:line 33
--- End of stack trace from previous location where exception was thrown ---
Hyperledger.Indy.InvalidStructureException : A value being processed is not valid.
```

**Comment by tmarkovski** [comment]
The unit test runner we have in CI uses libindy 1.12.0 and libsovtoken 1.0.3, as defined in this docker file at https://github.com/streetcred-id/docker image `streetcred/dotnet-indy:1.12.1`
I believe latest versions are libindy 1.13.0 and 1.0.5 for libsovtoken and libsovtokenfees. I try to update these occasionally, to make sure we're following on latest updates. The dependency for compatibility also includes running the correct version of indy-plenum. A lot of these streams move fast, so it's hard to follow them every time they release a version, so we only do periodic updates.

I wouldn't be too worried about this specific unit test, unless you are using sovrin tokens for your work.
Let's keep this issue open. It seems some structure changed between the indy node you're running and the sovtoken fees library.

**Comment by tmarkovski** [comment]
@jeffallen I have updated (#62) the test images to latest indy node, indy sdk and sovtoken versions.
indy-node 1.12.0, indy-sdk 1.14.2, and sovtoken 1.0.5. I haven't been able to reproduce the exception above. All unit tests seem to pass.
Are you still experiencing this issue?

**Comment by jeffallen** [comment]
I do not have my dotnet build environment working anymore. Closing.
